spec/setup.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Test setup for server-side tests.
+ */
+var chai = require("chai");
+var sinonChai = require("sinon-chai");
+
+// Add chai plugins.
+chai.use(sinonChai);
```

Comment:
We're not using it. I suppose I could pull it in when we do. I wanted to start us out with the "complete" stack but I'm also find to build out as we go. Up to you - I'm indifferent.